code/modules/power/singularity/narsie.dm

```diff
@@ -62,16 +59,15 @@
 	godsmack(A)
 	return
 
-/obj/singularity/narsie/proc/godsmack(var/atom/A)
-	if(istype(A,/obj/))
+/obj/singularity/narsie/proc/godsmack(var/atom/A) //you dare stand before a goddess?!
```

Comment:
Just go ahead and make it official that Nar-Sie is a female! You have been wanting it for months!